src/routes/transactions/entities/multisig-transaction.entity.ts

```diff
@@ -0,0 +1,41 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+
+export class ExecutionInfo {
+  @ApiProperty()
+  type: string;
+  @ApiProperty()
+  nonce: number;
+  @ApiProperty()
+  confirmationsRequired: number;
+  @ApiProperty()
+  confirmationsSubmitted: number;
+  @ApiPropertyOptional()
+  missingSigners?: string[];
```

Comment:
`missingSigners` are not really an `array` of strings, https://github.com/safe-global/safe-client-gateway/blob/main/src/routes/transactions/models/summary.rs#L155

Reply:
Changed in [85720bd](https://github.com/5afe/safe-client-gateway-nest/pull/197/commits/85720bdf33a94eaab2d886ffe53d1e0f35df0059)